packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -25,8 +25,22 @@ export default {
       }
     },
     stats: {
+      // 1 second intervals to collect stats data
       interval: 1000,
-      historyMax: 1800
+      // hold the last 2 minute of a calls data
+      historyMax: 120
+    },
+    metrics: {
+      // change to your client name else data will be muddled
+      // you do not need a specific format, and you do not need to register it
+      clientName: 'WEBEX_JS_SDK',
+      // TODO: for now this line has to be whitelisted, which is problematic for third party
+      clientType: 'TEAMS_CLIENT',
+      // send average values MQA in 60 second intervals
+      mqaMetricsInterval: 60000,
+      // send to cisco internal MQA data automatically
+      // we already send CA
+      autoSendMQA: true
```

Comment:
I think this should default to false? @adamweeks what's the stance on us collecting data by default from third party devs? Do we do this anywhere else?

Reply:
We send CA metrics by default, this was discussed with Matt months ago.

Reply:
I agree, it should default to false. Third parties should opt-in to data collection.

Reply:
Changing this value won't change anything now since code exists that will always send to CA. This will need to be addressed as tech debt going forward. Also it was signed off on previously by multiple levels so I want to make sure we have the time to discuss as a whole group. Changing that is out of scope in this PR.

Reply:
![img](https://media0.giphy.com/media/xT3i1iDlOembmemtdS/giphy.gif)